doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -194,7 +209,16 @@ a. Download and extract the Service OS kernel
    Reboot system. You should see the Grub menu with the new “ACRN ubuntu SOS”
    entry. Select it and proceed to booting the platform. The system will start
    the Ubuntu Desktop and you can now log in (as before).
-
+   
+   .. note::
+        if you are not seeing Grub menu after reboot system, you may to into a wrong boot entry
+        you may need to select “ACRN Hypervisor” manually by entering the EFI firmware at boot (using :kbd:`F10`)
+        
+   .. note::
+        for the first time reboot after install ACRN, you may see Black Screen after boot up succusfully 
```

Comment:
What GitHub issue number explains this?  The two I see don't indicate the blackscreen will just go away after a moment?

Reply:
there is no GitHub issue number,  did run test?  don't have too much people to run Ubuntu as the SOS at this moment,  i run and generate steps by myself with latest acrn release. and back screen can be reproduced,